extensions/positron-duckdb/src/extension.ts

```diff
@@ -1570,6 +1570,12 @@ export class DataExplorerRpcHandler {
 				const query = `CREATE OR REPLACE TABLE ${catalogName} AS
 				SELECT * FROM parquet_scan('${virtualPath}');`;
 				await this.db.runQuery(query);
+			} else if (baseExt === '.xlsx') {
+				await this.db.runQuery('INSTALL excel FROM core_nightly;');
```

Comment:
Since this "phones home" to load the extension from duckdb.org (http://nightly-extensions.duckdb.org, see https://duckdb.org/docs/stable/extensions/installing_extensions.html), it might fail in some environments where Positron isn't able to access the public internet, but that seems like an acceptable caveat until we sort out how to bundle supported DuckDB extensions with Positron to avoid installing from the internet.

Reply:
Yeah, good callout.